egs/multi_zh-hans/ASR/RESULTS.md

```diff
@@ -0,0 +1,30 @@
+## Results
+
+### WenetSpeech char-based training results (Non-streaming and streaming) on zipformer model
+
+This is the [pull request #1238](https://github.com/k2-fsa/icefall/pull/1238) in icefall.
+
+#### Non-streaming
+
+Best results (num of params : ~68M):
+
+The training command:
+
+```
+./zipformer/train.py \
+  --world-size 4 \
+  --num-epochs 23 \
+  --use-fp16 1 \
+  --max-duration 500 \
+  --num-workers 8
+```
+
+Character Error Rates (CERs) listed below are produced by the checkpoint of the 20th epoch using greedy search and BPE model ( # tokens is 2000, byte fallback enabled).
```

Comment:
Could you also list the decoding command?

What is the value of `--use-averaged-model` and `--avg`?